htdocs/salaries/class/paymentsalary.class.php

```diff
@@ -536,14 +569,26 @@ public function addPaymentToBank($user, $mode, $label, $accountid, $emetteur_nom
 					}
 				}
 
-				// Add link 'company' in bank_url between invoice and bank transaction (for each invoice concerned by payment)
-				$linkaddedforthirdparty = array();
-				foreach ($this->amounts as $key => $value) {
-					if ($mode == 'payment_salary') {
-						$salary = new Salary($this->db);
-						$salary->fetch($key);
-						$result = $acc->add_url_line($bank_line_id, $salary->id, DOL_URL_ROOT.'/salaries/card.php?id=', '('.$salary->label.')', 'salary');
```

Comment:
We should keep the link with the salary. Later we will be able to guess the user from this object.

Reply:
I'm not ok with you. In fact, you have alrealdy a link to payment salary in first column and the column Thirdparty in page bankentries is for thirdparty/user... not to have a link to salary. It's the same functionality than expense report module

![image](https://user-images.githubusercontent.com/2341395/127432604-6a2aa535-5d0b-4e84-9663-1e222a445492.png)

Reply:
You're right.
Let's add the add_url_line for user and removed the one on salary.
I pushed the fix manually because i introduced also other fixes.